buildSrc/subprojects/performance/src/main/groovy/org/gradle/testing/PerformanceTest.java

```diff
@@ -206,6 +206,7 @@ private void addExecutionParameters(List<String> result) {
             addSystemPropertyIfExist(result, "org.gradle.performance.execution.runs", runs);
             addSystemPropertyIfExist(result, "org.gradle.performance.execution.checks", checks);
             addSystemPropertyIfExist(result, "org.gradle.performance.execution.channel", channel);
+            addSystemPropertyIfExist(result, "org.gradle.ignoreBuildJavaVersionCheck", true);
```

Comment:
I don't understand why we need to pass the system property here. We always want to disable the build java version check for the performance tests we run.

Reply:
Yes you're right - I was mislead by `java9Home` and thought it needs to be set. Removed.